### DISCUSSION TITLE
For LFS is the limit 5GB per individual file, or 5GB of total stoage?

### DISCUSSION THREAD

**Discussion by brianvanvoorstBBN** [community/community#43765] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have found the documentation confusing on LFS.  On other pages disk space is discussed in terms of "file system storage", but when you reach the LFS table it says maximum "file size":

[https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-git-large-file-storage]

The title is my question:  Is LFS limited to 5GB total, or a set of files each of which is less than 5GB (I have my own guess as to what makes sense, but I'll wait for an official answer). 

Thank you.

**Comment by stefanzugal** [comment]
@brianvanvoorstBBN I see there is no answer here - but did you possibly find the answer on your own? We have the same issue, and I also cannot find any specific information on this topic. Also I do have my own guess, but I also would like to have an official answer for this.

**Comment by stefanzugal** [comment]
@brianvanvoorstBBN I have contacted GitHub support - their answer:

> I've had a look into this for you and as LFS doesn't count towards the repo size limit, you are welcome to buy as many data packs and store as many LFS objects as you wish!

That makes pretty much sense, and I guess you will also have assumed that. So here's the official answer as well.
